registry/verify-namespace-owner/verify_namespace_owner.go

```diff
@@ -99,6 +100,11 @@ func parseConfig(tk toolkit.Toolkit) (config, error) {
 		return config{}, toolkit.FailedError("namespace must be set")
 	}
 
+	c.ListNamespaces, ok = tk.GetInputList([]string{"list-of-namespaces"})
```

Comment:
Let's renamed this to `blocked-namespaces`

Reply:
In the latest commit, I have tried to change this `blockedNamespaces`, should it work?

Reply:
yea, but I think you'll also want to change the `"list-of-namespaces"` to `"blocked-namespaces"`

Reply:
Oh, my bad, understood, will change that

Reply:
Done that, thanks a lot!